### ISSUE TITLE
Undefined invisibility type in lang file

### ISSUE THREAD

**Issue by wodim** [Sarjuuk/aowow#247]
https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=60197

    Lang - undefined property Lang::$spell['invisibilityType']['8']

Maybe it was added recently to Trinity? Would be nice to have.

**Comment by wodim** [comment]
Also 7 here:

https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=60194

**Comment by Sarjuuk** [comment]
```
enum InvisibilityType
{
    INVISIBILITY_GENERAL     =  0,
    INVISIBILITY_UNK1        =  1,
    INVISIBILITY_UNK2        =  2,
    INVISIBILITY_TRAP        =  3,
    INVISIBILITY_UNK4        =  4,
    INVISIBILITY_UNK5        =  5,
    INVISIBILITY_DRUNK       =  6,
    INVISIBILITY_UNK7        =  7,
    INVISIBILITY_UNK8        =  8,
    INVISIBILITY_UNK9        =  9,
    INVISIBILITY_UNK10       = 10,
    INVISIBILITY_UNK11       = 11,

    TOTAL_INVISIBILITY_TYPES = 12
};
```
no idea.
will set them to null to suppress the error.

**Comment by Sarjuuk** [comment]
fixed in 4ed2d0836f21385536ae76e3eaf0fdec3cbec6c0
